src/manual-multiple/web/index.html

```diff
@@ -72,22 +72,23 @@ <h2 class="blink__anchor advert__title">[%Offer4Title%]</h2>
     </div>
 </aside>
 <script>
+    const genericLogoTones = ['brand', 'lifestyle', 'climate', 'climate2', 'support']
+    const subscriptionsLogoTones = ['subscription', 'subs-rebrand']
+    
+    const genericLogoString = '{{#svg}}guardian-generic-logo{{/svg}}'
+    const subscriptionsLogoString = '{{#svg}}guardian-subscriptions-logo{{/svg}}'
+
     var logoSvgs = {
         'job': '{{#svg}}guardian-jobs-logo{{/svg}}',
         'live': '{{#svg}}guardian-live-logo{{/svg}}',
         'travel': '{{#svg}}guardian-holidays-logo{{/svg}}',
         'money': '{{#svg}}guardian-moneydeals-logo{{/svg}}',
         'book': '{{#svg}}guardian-bookshop-logo{{/svg}}',
         'masterclass': '{{#svg}}guardian-masterclasses-logo{{/svg}}',
-        'subscription': '{{#svg}}guardian-subscriptions-logo{{/svg}}',
-        'brand': '{{#svg}}guardian-generic-logo{{/svg}}',
         'weekly': '{{#svg}}guardian-weekly-logo{{/svg}}',
         'members': '{{#svg}}guardian-members-logo{{/svg}}',
         'patron': '{{#svg}}guardian-patron-logo{{/svg}}',
-        'lifestyle': '{{#svg}}guardian-generic-logo{{/svg}}',
-        'climate': '{{#svg}}guardian-generic-logo{{/svg}}',
-        'climate2': '{{#svg}}guardian-generic-logo{{/svg}}',
-        'support': '{{#svg}}guardian-generic-logo{{/svg}}',
-        'subs-rebrand': '{{#svg}}guardian-subscriptions-logo{{/svg}}',
+        ...Object.fromEntries(genericLogoTones.map(tone => [tone, genericLogoString])),
+        ...Object.fromEntries(subscriptionsLogoTones.map(tone => [tone, subscriptionsLogoString]))
```

Comment:
Considering there’s [no support for Object.entries in IE][caniuse], we could also simply make it a clean object like so:

```suggestion
        'brand': genericLogoString,
        'lifestyle': genericLogoString,
        'climate': genericLogoString,
        'climate2': genericLogoString,
        'support': genericLogoString,
        'subscription': subscriptionsLogoString,
        'subs-rebrand': subscriptionsLogoString,
```

[caniuse]: https://caniuse.com/?search=Object.entries

Reply:
I hadn't thought about compatibility - you're absolutely right. 
The build size is also not affected by this change, which is great, and it's also much better in terms of code readability.